deployer/src/deployment/queue.rs

```diff
@@ -453,13 +453,13 @@ async fn run_pre_deploy_tests(
 /// This will store the path to the executable for each runtime, which will be the users project with
 /// an embedded runtime for alpha, and a .wasm file for shuttle-next.
 #[instrument(skip(executable_path, to_directory, new_filename))]
-async fn move_executable(
+async fn copy_executable(
```

Comment:
What is this change addressing?

Reply:
Adhoc redeployment speed improvement. Should cherry pick it to another branch tbh.